src/views/app/Dashboard/Index.vue

```diff
@@ -1,38 +1,73 @@
 <script setup lang="ts">
-import { watch } from 'vue';
+import { computed, watch } from 'vue';
 
 import useSalesStore from '@/store/sales';
 import useRestaurantStore from '@/store/restaurant';
-import SalesGraph from '@/components/Graphs/SalesGraph.vue';
 import GlobalLoading from '@/components/GlobalLoading.vue';
+import SalesGraph from '@/components/Graphs/SalesGraph.vue';
+import DetailCard from '@/views/app/Dashboard/components/DetailCard.vue';
+import { formatPriceToDisplay } from '@/utils/inputFormats';
 
 const salesStore = useSalesStore()
 const restaurantStore = useRestaurantStore();
 
-async function getSalesPerMonth(restaurantId: string): Promise<void> {
+const hasDetailData = computed(() => salesStore.currentMonthRevenue && salesStore.ordersOpen && salesStore.salesMonthClosed)
+const currentRevenue = computed(() => formatPriceToDisplay(String(salesStore.currentMonthRevenue)));
+
+async function getDashboardData(restaurantId: string): Promise<void> {
   if (!salesStore.salesPerMonth) {
     await salesStore.getSalesPerMonth(restaurantId);
   }
+  if (!hasDetailData.value) {
+    await salesStore.getSalesCurrentMonth(restaurantId);
+  }
```

Comment:
add a ";"

Reply:
a block in javascript does not use semicolon

Reply:
ñiñiñi